ReSwiftTests/AutomaticallySkipRepeatsTests.swift

```diff
@@ -28,19 +28,19 @@ class AutomaticallySkipRepeatsTests: XCTestCase {
     }
 
     func testInitialSubscription() {
-        store.subscribe(self) { $0.select { $0.name } }
+        store.subscribe(self) { $0.select(\.name) }
```

Comment:
can you duplicate these tests instead of replacing the old ones? Then we have a way to ensure that both implementations do the same and catch regressions